native/tools/build-config

```diff
@@ -19,7 +19,11 @@ const loadBuildConfigAsXCConfig = buildConfigName => {
     delimiter: '_',
     transformKey: key => decamelize(key).toUpperCase()
   })
-  const prefixed = Object.keys(xcconfigOptions).map(key => `BUILD_CONFIG_${key} = ${xcconfigOptions[key]}`)
+  const prefixed = Object.keys(xcconfigOptions).map(key => {
+    const value = xcconfigOptions[key]
+    const escaped = typeof xcconfigOptions[key] === 'string' ? value.replace(/\n/g, '\\n') : value
```

Comment:
why do we need to do that?

Reply:
Now specifically: Because of the imprint (which has multiple line breaks)
In general: xcconfig files are key value files with one key/value per line. If there are linebreaks in the value, the next line is treated as another key value pair which it obviously is not, which leads to errors.

Reply:
isn't `typeof xcconfigOptions[key] === 'string'` always true?

Reply:
No, there are e.g. boolean values as well (feature flags for example).

Reply:
Maybe use lodashs function then. I dont unterstand the implementation but probably it is better: https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L12135